### PR TITLE
Jonfritz patch 3 docs update

### PR DESCRIPTION
**Pull request by jonfritz**
Update main landing page

### CODE REVIEW

File: docs/source/images/ArynArchitecture_APS+Sycamore.png
Comment (HenryL27) [inline docs/source/images/ArynArchitecture_APS+Sycamore.png:1]: can we put pdf in the input list to sycamore and a bidirectional arrow between sycamore and APS?

Comment (jonfritz) [reply docs/source/images/ArynArchitecture_APS+Sycamore.png:1]: Ack. Will do and update.

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,88 +1,89 @@
...
+You will need an Aryn Cloud API key, which you can get for free `here <aryn.ai/get-started>`_. You will recieve the API key in your email inbox.
 
-``git clone https://github.com/aryn-ai/sycamore``
+Next, you can:
 
-2.	Set OpenAI Key:
+**Use the Aryn Playground:** Visit `the Playground <https://play.aryn.cloud/partitioning>`_ and use the UI to see how the service segments, lables, and extracts data from your documents.
 
-``export OPENAI_API_KEY=YOUR-KEY``
+|
```
Comment (HenryL27) [inline docs/source/index.rst:38]: what does this | do?

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,88 +1,89 @@
...
-``http://localhost:3000/``
+2. Then, partition your document:
 
-You can next choose to run a demo that :doc:`prepares and ingests data from the Sort Benchmark website </welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset>`, :doc:`crawl data from a public website <welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website>`, or :doc: `write your own data preparation script </data_ingestion_and_preparation/using_jupyter>`.
+.. code-block:: python
 
-For more info about Sycamore’s data ingestion and preparation feature set, visit the :doc:`Sycamore documentation </data_ingestion_and_preparation/data_preparation_concepts>`.
+    from aryn_sdk.partition import partition_file
+    with open("mydocument.pdf", "rb") as f:
+    data = partition_file(f)
```
Comment (HenryL27) [inline docs/source/index.rst:56]: indent `data = partition_file(f)`

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,88 +1,89 @@
...
+.. code-block:: python
 
-``docker compose run crawl_http http://my.website.example.com``
+    pip install sycamore-ai
 
-c. Write :doc:`custom data ingestion and preparation code using the Jupyter container </data_ingestion_and_preparation/using_jupyter>`. Access it via the URL from:
+..
 
-``docker compose logs jupyter | grep Visit``
+2. You can next Sycamore scripts to process your docuemnts and unstructured data. `This notebook <https://github.com/aryn-ai/sycamore/blob/main/notebooks/metadata-extraction.ipynb>`_ demonstrates a variety of Sycamore data transforms and loads an OpenSearch hybrid search index.
```
Comment (HenryL27) [inline docs/source/index.rst:75]: typos

**Review by HenryL27 — APPROVED** [review]
basically just typos

File: docs/source/index.rst [hunk 4]
```diff
@@ -1,88 +1,89 @@
...
+The Aryn Partitioning Service (APS) is a serverless, GPU-powered API for segmenting and labeling PDF documents, doing OCR, and extracting tables and images. It returns the output in JSON. APS runs the Aryn Partitioner and it’s `state-of-the-art, open source deep learning DETR AI model <https://huggingface.co/Aryn/deformable-detr-DocLayNet>`_ trained on 80k+ enterprise documents. You can use it to partition documents and extract information directly in your code, or use with Sycamore for additional processing. `Sign-up here for free <https://www.aryn.ai/get-started>`_ to get an API Key and use the `Aryn Playground <https://play.aryn.cloud/partitioning>`_ to visually see how it segments and processes your own documents. Or, watch the Aryn Partitioning Service in action in `this video <https://www.aryn.ai/?name=ArynPartitioningService_Intro>`_.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/index.rst:5]: Ack

File: docs/source/index.rst [hunk 5]
```diff
@@ -1,88 +1,89 @@
...
 
+Sycamore is a document processing engine licensed covered under the Apache v2.0 license. It's built for complex unstructured data, such as documents, presentations, transcripts, embedded tables, and internal knowledge repositories. Sycamore provides a declarative dataflow abstraction called DocSets to make manipulating unstructured documents easy and scalable. It’s similar in style to Apache Spark and Pandas, but for collections of unstructured documents. DocSets can be used not only for extracting, enriching, summarizing, and cleaning unstructured data, but also for running powerful analytics on these datasets.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/index.rst:7]: Ack

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,88 +1,89 @@
...
 
-.. image:: images/SycamoreDiagram2.png
+.. image:: images/ArynArchitecture_APS%2BSycamorev2.png
 
 **Key Features**
 
-* **Answer Hard Questions on Complex Data.** Prepares and enriches complex unstructured data for search and analytics through advanced data segmentation, LLM-powered UDFs for data enrichment, performant data manipulation with Python, and vector embeddings using a variety of AI models.
-
-* **Multiple Query Options.** Flexible query operations over unstructured data including RAG, hybrid search, analytical functions, natural language/conversational search, and custom post-processing functions.
+* **More powerful and accurate document segmentation for complex documents with tables, images, text, infographics, and more.**  Aryn’s vision-based segmentation models provide 11+ class labels, and are up to 6x better in mean average precision (mAP) and 4x better in mean average recall (mAR) than existing solutions.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/index.rst:15]: Me neither

File: docs/source/index.rst (hunk 2 above)
Comment (jonfritz) [reply docs/source/index.rst:56]: Ack

File: docs/source/index.rst (hunk 3 above)
Comment (jonfritz) [reply docs/source/index.rst:75]: Ack

**Pull request merged**